app/serializable/serializable_submission_entry.rb

```diff
@@ -7,6 +7,8 @@ class SerializableSubmissionEntry < JSONAPI::Serializable::Resource
   attribute :source
   attribute :data
 
+  attribute :validation_errors
```

Comment:
Unless I'm mistaken, this seems to be unrelated to the rest of the commit?

Reply:
It is used during checks for validation in the frontend code.

Reply:
Fair enough :+1: